src/coreclr/gc/gc.cpp

```diff
@@ -12643,6 +12643,7 @@ void gc_heap::distribute_free_regions()
     size_t num_decommit_regions_by_time = 0;
     size_t size_decommit_regions_by_time = 0;
     size_t heap_budget_in_region_units[MAX_SUPPORTED_CPUS][kind_count];
+    size_t min_heap_budget_in_region_units[MAX_SUPPORTED_CPUS][kind_count];
```

Comment:
since we are only using this for SOH, it seems like this can just be declared as 

`size_t min_heap_budget_in_region_units[MAX_SUPPORTED_CPUS];`

MAX_SUPPORTED_CPUS is 1024 so this is quite a large data structure.

Reply:
Well, we are using it for UOH as well, but it's always 0 in this case, so we can use an if-statement or conditional expression.

I'll fix it...